docker/scripts/make-lockfile.R

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env Rscript
+
+# This script is used to make a lockfile based on the current renv environment,
```

Comment:
This is updating a lockfile correct? It's not actually creating a new one.

Reply:
I think it should create a new one if it did not exist. I will change to "create or update"